src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,29 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+    if (typeof text !== "string")
+        throw ("Please enter a valid string");
```

Comment:
Throw `Error`, not string. Throwing `Error` would preserve the stack trace, useful for debugging.